dagster_polars/io_managers/parquet.py

```diff
@@ -171,55 +148,118 @@ def downsteam(upstream: DataFrameWithMetadata):
     """
 
     extension: str = ".parquet"  # type: ignore
-    use_legacy_reader: bool = False
 
-    def dump_df_to_path(
+    def sink_df_to_path(
         self,
         context: OutputContext,
-        df: pl.DataFrame,
+        df: pl.LazyFrame,
         path: "UPath",
         metadata: Optional[StorageMetadata] = None,
     ):
         assert context.metadata is not None
 
-        table: Table = df.to_arrow()
-
         if metadata is not None:
-            existing_metadata = table.schema.metadata.to_dict() if table.schema.metadata is not None else {}
-            existing_metadata.update({DAGSTER_POLARS_STORAGE_METADATA_KEY: json.dumps(metadata)})
-            table = table.replace_schema_metadata(existing_metadata)
-
+            context.log.log(
```

Comment:
why not `context.log.warning`?

Reply:
No reason, changed it to context.log.warning